api/src/lib/aws.ts

```diff
@@ -117,8 +119,12 @@ export async function s3PutSignedUrl(
 }
 
 export async function deleteUploadFile(upload: Upload) {
-  const fileKey = `uploads/${upload.agency.organizationId}/${upload.agencyId}/${upload.reportingPeriodId}/${upload.id}/${upload.filename}`
-  await s3DeleteObject(fileKey)
+  try {
+    const fileKey = `uploads/${upload.agency.organizationId}/${upload.agencyId}/${upload.reportingPeriodId}/${upload.id}/${upload.filename}`
+    await s3DeleteObject(fileKey)
+  } catch (error) {
+    logger.error(error, 'Error deleting file from S3:')
```

Comment:
Minor nit: the `:` at the end of the log message is unnecessary since logs are structured and might not be displayed as `the log message: the error message` as a single string. I mention this since having it might make the log more difficult to query for in Datadog.